### ISSUE TITLE
Appcast not worked with utf-8 correctly. 

### ISSUE THREAD

**Issue by krunduev** [larryaasen/upgrader#111]
For example Russian released notes is not showed correctly.

Fix:

appcast.dart

Add
import 'dart:convert' show utf8;

Change
    final contents = response.body;
To
    final contents = utf8.decode(response.bodyBytes);

**Comment by larryaasen** [comment]
@krunduev  Thanks for reporting this issue. Do you have a small sample text that is broken that I could use for a unit test?

**Comment by larryaasen** [comment]
PR to fix this issue: https://github.com/larryaasen/upgrader/pull/112

**Comment by krunduev** [comment]
If we get
```
<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0" xmlns:sparkle="http://www.andymatuschak.org/xml-namespaces/sparkle">
    <channel>
        <title>Тестовое приложение</title>
        <item>
            <title>Версия 3.1.5</title>
            <description>- исправлены некоторые ошибки</description>
            <pubDate>Sun, 10 Jun 2021 12:00:00 +0000</pubDate>
            <enclosure url="https://play.google.com/store/apps/details?id=com.test" sparkle:version="3.1.5" sparkle:os="android" />
        </item>
    </channel>
</rss>
```
`final contents = response.body;
`
gives
```
<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0" xmlns:sparkle="http://www.andymatuschak.org/xml-namespaces/sparkle">
    <channel>
        <title>Ð¥Ð¸ÑÐ°Ñ - Ð¢Ð¾ÑÐ°,Ð¢Ð°Ð½Ð¸Ñ,Ð¢ÐµÐ³Ð¸Ð»Ð¸Ð¼</title>
        <item>
            <title>ÐÐµÑÑÐ¸Ñ 3.1.5</title>
            <description>- Ð¸ÑÐ¿ÑÐ°Ð²Ð»ÐµÐ½Ñ Ð½ÐµÐºÐ¾ÑÐ¾ÑÑÐµ Ð¾ÑÐ¸Ð±ÐºÐ¸</description>
            <pubDate>Sun, 10 Jun 2021 12:00:00 +0000</pubDate>
            <enclosure url="https://play.google.com/store/apps/details?id=com.goldin.hitas" sparkle:version="3.1.5" sparkle:os="android" />
        </item>
    </channel>
</rss>
```

`    final contents = utf8.decode(response.bodyBytes);
`
Works as expected

**Comment by larryaasen** [comment]
Thanks. I used some of that for a unit test.

**Comment by larryaasen** [comment]
This issue was fixed with the release of this version: [3.3.1](https://github.com/larryaasen/upgrader/releases/tag/3.3.1)
